test/e2e_node/cpu_manager_test.go

```diff
@@ -1968,24 +1961,35 @@ var _ = SIGDescribe("CPU Manager Incompatibility Pod Level Resources", ginkgo.Or
 	ginkgo.BeforeEach(func(ctx context.Context) {
 		// note intentionally NOT set reservedCPUs -  this must be initialized on a test-by-test basis
 		podMap = make(map[string]*v1.Pod)
+		reservedCPUs = cpuset.New()
 	})
 
 	ginkgo.JustBeforeEach(func(ctx context.Context) {
 		if !e2enodeCgroupV2Enabled {
 			e2eskipper.Skipf("Skipping since CgroupV2 not used")
 		}
+		// use a closure to minimize the arguments, to make the usage more straightforward
+		skipIfAllocatableCPUsLessThan = func(node *v1.Node, val int) {
+			ginkgo.GinkgoHelper()
+			checkAllocatableCPUs(node, val, reservedCPUs, onlineCPUs, smtLevel)
+		}
 	})
 
 	ginkgo.AfterEach(func(ctx context.Context) {
 		deletePodsAsync(ctx, f, podMap)
 	})
 
 	ginkgo.When("running guaranteed pod level resources tests", ginkgo.Label("guaranteed pod level resources", "reserved-cpus"), func() {
-		ginkgo.It("should let the container access all the online CPUs without a reserved CPUs set", func(ctx context.Context) {
+		ginkgo.It("scope: pod, should allocate exclusive CPUs to a guaranteed pod with pod-level resources and guaranteed container, PodLevelResourceManagers enabled", func(ctx context.Context) {
+			skipIfAllocatableCPUsLessThan(getLocalNode(ctx, f), 1)
+
 			updateKubeletConfigIfNeeded(ctx, f, configureCPUManagerInKubelet(oldCfg, &cpuManagerKubeletArguments{
-				policyName:              string(cpumanager.PolicyStatic),
-				reservedSystemCPUs:      cpuset.CPUSet{},
-				enablePodLevelResources: true,
+				policyName:                     string(cpumanager.PolicyStatic),
+				topologyManagerPolicy:          topologymanager.PolicyBestEffort,
```

Comment:
why is the topology manager relevant again and why should we not depend on the default? (same applies for memory manager e2e tests)